pv_vardict_0.0.1/case-control_filter_0.0.1.cwl

```diff
@@ -0,0 +1,113 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+baseCommand:
+  - pv
+inputs:
+  - id: memory_per_job
+    type: int?
+    doc: Memory per job in megabytes
+  - id: memory_overhead
+    type: int?
+    doc: Memory overhead per job in megabytes
+  - id: number_of_threads
+    type: int?
+  - id: inputVCF
+    type: File
+    inputBinding:
+      position: 0
+      prefix: '-i'
+      shellQuote: false
+    doc: Input vardict vcf to be filtered
+  - id: tsampleName
+    type: string
+    inputBinding:
+      position: 1
+      prefix: '--tsampleName'
+      shellQuote: false
+    doc: Name of the tumor Sample
+  - id: alleledepth
+    type: int
+    inputBinding:
+      position: 2
+      prefix: '-ad'
+      shellQuote: false
+  - id: totalDepth
+    type: int?
+    inputBinding:
+      position: 2
+      prefix: '-dp'
+      shellQuote: false
+    doc: Tumor total depth threshold
+  - id: tnRatio
+    type: int?
+    inputBinding:
+      position: 2
+      prefix: '-tnr'
+      shellQuote: false
+    doc: Tumor-Normal variant fraction ratio threshold
+  - id: variantFraction
+    type: float?
+    inputBinding:
+      position: 2
+      prefix: '-vf'
+      shellQuote: false
+    doc: Tumor variant fraction threshold
+  - id: minQual
+    type: int?
+    inputBinding:
+      position: 2
+      prefix: '-mq'
+    doc: Minimum variant call quality
+  - id: filterGermline
+    type: boolean?
+    inputBinding:
+      position: 2
+      prefix: '-fg'
+    doc: Whether to remove calls without 'somatic' status
+outputs:
+  - id: txt
+    type: File
+    outputBinding:
+      glob: '*_STDfilter.txt'
+  - id: vcf
+    type:
+      - type: array
+        items: File
+    outputBinding:
+      glob: '*.vcf'
+arguments:
+  - vardict
+  - case-control
+  - filter
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: 40000
+    coresMin: 4
+  - class: InlineJavascriptRequirement
+hints:
+  - class: DockerRequirement
+    dockerPull: 'ghcr.io/msk-access/postprocessing_variant_calls:latest'
```

Comment:
We should use a specific version of the release and not the latest and I think we should make sure you make the release of the  postprocessing_variant_calls and then use that tag of the docker image

Reply:
fix in this commit: https://github.com/msk-access/cwl-commandlinetools/pull/148/commits/be5e53e5c5b8838222b3b1f477a0785684f4dab0 !